lib/sufia/engine.rb

```diff
@@ -1,4 +1,12 @@
 module Sufia
+  class Statefile
+    def self.default
+      return '/tmp/minter-state' if Rails.env.development? || Rails.env.test?
+      raise NotImplementedError, "can't use the default statefile for production" unless Dir.exist?('/var/sufia')
```

Comment:
Oh I see what you're saying... even if I have specified my own location, it would still execute `Sufia::Statefile.default`? I wonder if it can check to see if a value has been provided.